docs/building-extensions/plugins/plugin-events/content.md

```diff
@@ -0,0 +1,140 @@
+---
+sidebar_position: 1
+title: Content Plugin Events
+---
+
+Content Plugin Events
+=====================
+
+Content events are triggered during the content creation process. These events are triggered in many different components and modules - they are not specific to the `com_content` component. 
+
+This list gives a brief description of each event, what their parameters are, and any examples of their use which are available in the Joomla Manual.
+
+Where a Return Value is expected, this should be returned via the event `addResult` method, as described in [Joomla 4 and 5 changes](../joomla-4-and-5-changes.md).
+
+## Content\ContentPrepareEvent / onContentPrepare
+
+### Description
+
+This is the first stage in preparing content for output and is the most common point for content-orientated plugins to do their work. 
+Since the item and related parameters are passed by reference, event handlers can modify them prior to display. 
+
+### Parameters
+
+- **`context`** - The context of the content being passed to the plugin. This is the component name and view - or name of module (e.g. com_content.article, com_contact.contact, com_users.user). Use this to check whether you are in the desired context for the plugin.
+
+- **`&item`** - A reference to the item which is being rendered by the view, for example, an article, contact or user. 

```

Comment:
Just a note:
All this `referencing` will be need to update at some point later, we do not use it anymore.

Also whole `Parameters` section will be need to replace to "Event attributes", or something kind of that, similar to 
https://manual.joomla.org/docs/building-extensions/plugins/editors-xtd-plugin#oneditorbuttonssetup

I know it is copy from old page, so for now it is fine as it is, for beginning.